benchmark/infrastructure/BenchmarkMeasurementContainer.cpp

```diff
@@ -364,4 +370,36 @@ std::ostream& operator<<(std::ostream& os, const ResultGroup& resultGroup) {
   return streamInserterOperatorImplementation(os, resultGroup);
 }
 
+// ____________________________________________________________________________
+template <ad_utility::isTypeAnyOf<ResultEntry, ResultTable> T>
+requires std::same_as<std::decay_t<T>, T>
+void ResultGroup::deleteEntryImpl(T& entry) {
+  // The vector, that holds our entries.
+  auto& vec{std::invoke([this]() -> auto& {
+    if constexpr (std::same_as<T, ResultEntry>) {
+      return resultEntries_;
+    } else if constexpr (std::same_as<T, ResultTable>) {
+      return resultTables_;
+    } else {
+      AD_FAIL();
+    }
```

Comment:
`else {static_assert(std::same_as<T, ResultTable>); return resultTables_;}`
(better code coverage + the check is static.

Reply:
Great idea! Done in [6b7ac8c](https://github.com/ad-freiburg/qlever/pull/1207/commits/6b7ac8cf0a4b54a3eb50f3d6d4f17598ee792766).